### PR TITLE
Integration Test - Implement the validation for metric value availability and duplicate EMF logs

### PR DESCRIPTION
**Pull request by Paramadon**

### CODE REVIEW

File: terraform/eks/daemon/main.tf [hunk 1]
```diff
@@ -408,12 +408,24 @@ resource "kubernetes_cluster_role_binding" "rolebinding" {
...
+  create_duration = "2m"
+}
+
 resource "null_resource" "validator" {
   depends_on = [
     aws_eks_node_group.this,
     kubernetes_daemonset.service,
     kubernetes_cluster_role_binding.rolebinding,
     kubernetes_service_account.cwagentservice,
+    time_sleep.wait_2_min
```
Comment (sethAmazon) [inline terraform/eks/daemon/main.tf:428]: Why do we need to wait?

**Comment by sethAmazon** [comment]
Can you run this against the full sweet to make sure you did not break any other tests since it seems you touched common code.

File: terraform/eks/daemon/main.tf (hunk 1 above)
Comment (Paramadon) [reply terraform/eks/daemon/main.tf]: removed and it still passes all tests

File: test/metric/metric_value_query.go [hunk 2]
```diff
@@ -51,7 +51,7 @@ func (n *MetricValueFetcher) Fetch(namespace, metricName string, metricSpecificD
 	}
 
 	endTime := time.Now()
-	startTime := subtractMinutes(endTime, 10)
+	startTime := subtractMinutes(endTime, 2)
```
[the remark this answers is not in the document]
Comment (Paramadon) [reply test/metric/metric_value_query.go]: Changed it back(this was part of a different fork) and integration test still works (added another picture of other tests passing)

**Review by sethAmazon — APPROVED** [review]

**Review by ymtaye — APPROVED** [review]

**Pull request merged**